README.md

```diff
@@ -1,42 +1,144 @@
 * Explain that a function has a name that identifies what it does, and that name is used to "call" the function to perform its task when needed.
 * Create functions that contain multiple arguments
 
-
 ## What the student can do at this point 
 
 * Create variables and constants
-* Knows how to use the print function (although they might not know that it's referred to as a function at this point).
-* Can create variables of type String and Int (no other types).
-* They would have learned what functions are and will be just coming off having completed a functions lab in Playground.
-* They would have completed a lab in a playground file which had them work with the following - String, Int, variables, constants, print, string interpolation, type inference, type annotations.
+* Knows how to use the `print` function (although they might not know that it's referred to as a function at this point).
+* Can create variables of type `String` and `Int`.
+* They would have learned what functions are and will be just coming off having completed a functions lab.
+* They would have completed a lab in a playground file which had them work with the following: `String`s, `Int`s, variables, constants, `print()`, string interpolation, type inference, type annotations.
+
+## Functions and Arguments
+
+In the last lesson, you learned how to create your own functions to keep self-contained chunks of code in one place, and call them as often as you want. You even learned how to create a function that takes an argument, so you can generalize its behavior a bit. Hopefully you saw how useful functions can be when writing code in Swift, but now you're about to see how they can be even _more_ useful.
+
+Let's start, though, by reviewing how you create functions.
+
+Here's your goal: Write a function that will print the string "There are &lt;number&gt; moons orbiting &lt;planet&gt;." This will let you show off to our friend Neil DeGrasse Tyson how much you know about astronomy, _and_ write some Swift code in the meantime. Start by proving how much you know about Jupiter (hint: Jupiter has _67_ moons).
+
+![67 moons, ah ah ah](.images/count.jpg)
+
+What's the simplest way to write your function?
+
+A simple way to write a function to print "There are 67 moons orbiting Jupiter." would be this:
+
+```swift
+func moonsOfJupiter() {
+    print("There are 67 moons orbiting Jupiter.")
+}
+```
+
+Take a moment to remember how you write functions in Swift. Functions are introduced using the `func` keyword. Then, you write the name of the function—in this case, `moonsOfJupiter` (remember, function names can't contain spaces!). Next comes the argument list in parentheses. In this case, `moonsOfJupiter` doesn't take any arguments, so you just write an empty set of parentheses. Then you write the body of the function—the statements that will be executed when the function is called—between curly braces.
+
```

Comment:
Love it. I like reinforcing this here to help them solidify their knowledge on functions.